pkg/authorization/authorizer/subjects_test.go

```diff
@@ -1,80 +0,0 @@
-package authorizer
-
-import (
-	"testing"
-
-	"k8s.io/apimachinery/pkg/util/sets"
-	kauthorizer "k8s.io/apiserver/pkg/authorization/authorizer"
-
-	authorizationapi "github.com/openshift/origin/pkg/authorization/apis/authorization"
-	testpolicyregistry "github.com/openshift/origin/pkg/authorization/registry/test"
-	"github.com/openshift/origin/pkg/authorization/rulevalidation"
-)
-
-type subjectsTest struct {
-	policies              []authorizationapi.Policy
-	bindings              []authorizationapi.PolicyBinding
-	clusterPolicies       []authorizationapi.ClusterPolicy
-	clusterBindings       []authorizationapi.ClusterPolicyBinding
-	policyRetrievalError  error
-	bindingRetrievalError error
-
-	attributes kauthorizer.AttributesRecord
-
-	expectedUsers  sets.String
-	expectedGroups sets.String
-	expectedError  string
-}
-
-func TestSubjects(t *testing.T) {
```

Comment:
Is "this" a generic this ? Or specific to something that is missing from upstream rbac tests in something like this:
https://github.com/openshift/origin/blob/master/vendor/k8s.io/kubernetes/pkg/registry/rbac/validation/rule_test.go